src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -436,6 +436,13 @@ public async Task FireReminderAsync(ActorReminder reminder)
 
             try
             {
+                ActorTrace.Source.WriteInfoWithId(
+                    TraceType,
+                    this.traceId,

```

Comment:
Add log to emit time taken to load reminders in the context of runasync.

Reply:
I have added traces at the start and the end of the ActorManager.LoadRemindersAsync method (this is called by ActorService.RunAsync when it is trying to load the reminders). We can get the total time taken to load the reminders by these 2 traces.